app/models/tool_draft.rb

```diff
@@ -35,4 +37,22 @@ def display_short_name
   def display_entry_title
     entry_title || '<Untitled Tool Record>'
   end
+
+  private
+
+  def set_metadata_specification
+    # this is a hidden fieldset added to UMM-T to document the metadata version
+    # and specs that should be autopopulated
+    # TODO: we should try to populate these values from the schema enums
+    # MetadataSpecification
+    metadata_specification = {
+      'URL' => 'https://cdn.earthdata.nasa.gov/umm/tool/v1.0',
+      'Name' => 'UMM-T',
+      'Version' => '1.0'
+    }
+
+    unless self.draft['MetadataSpecification'] == metadata_specification
```

Comment:
x = y unless x == y

Reply:
I had that originally but it seemed a bit too long of a line.